tasks/vm.py

```diff
@@ -80,16 +80,14 @@ def _show_connection_message(ctx: Context, full_stack_name: str):
     )
 
 
-@task(
-    help={
-        "stack_name": doc.stack_name,
-    }
-)
-def destroy_vm(ctx: Context, stack_name: Optional[str] = None):
+@task(help={"stack_name": doc.stack_name, "yes": doc.yes})
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
What about using `force-yes` instead ?
**💬 suggestion**
```suggestion
@task(help={"stack_name": doc.stack_name, "force_yes": doc.yes})
```